.github/workflows/install.yml

```diff
@@ -75,6 +75,44 @@ jobs:
       run: sudo ./scripts/install.sh --debug
     - name: Test CLI
       run: doppler --version
+  ubuntu-force-verify-signature:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        path: ./src/github.com/${{ github.repository }}
+    - name: Install CLI
+      run: sudo ./scripts/install.sh --debug --verify-signature
+    - name: Test CLI
+      run: doppler --version
+  ubuntu-force-verify-signature-fail:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        path: ./src/github.com/${{ github.repository }}
+    - name: Uninstall gnupg
+      run: |
+        sudo apt-get remove -y gnupg;
+        sudo mv /usr/bin/gpg /usr/bin/gpg.old
+    - name: Verify gnupg has been removed
+      run: |
+        result=$(which gpg) || true;
+        if [ -n "$result" ]; then
+          exit 1;
+        fi;
+    - name: Install CLI
+      continue-on-error: true
+      run: sudo ./scripts/install.sh --debug --verify-signature
+    - name: Verify install failed
+      if: ${{ always() }}
```

Comment:
Just curious, why is this? :)

Reply:
The previous step is expected to fail, so we need this to avoid exiting.